src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -221,7 +226,19 @@ const StakingWidget = ({
      */
     userActivatedTokens > 0;
 
-  return (
+  /*
+   * Motion can be still staked (ie: amount left to stake)
+   */
+  const canUserStakeYay =
+    canUserStake && remainingToStakeSafe(remainingToStakeYay) > 0;
+  const canUserStakeNay =
+    canUserStake && remainingToStakeSafe(remainingToStakeNay) > 0;
+
+  const canBeStaked = isObjection ? canUserStakeNay : canUserStakeYay;
+
+  return !bigNumberify(totalNAYStakes).isZero() ? (
+    <div>PLACEHOLDER FOR ARMANDOS COMPONENT</div>
```

Comment:
Armando's component ? This should be the staking (yay side) slider, which I created

![Screenshot from 2021-04-27 23-19-33](https://user-images.githubusercontent.com/1193222/116307577-2c57d080-a7af-11eb-8d05-7b2fd07fe6e7.png)

Reply:
But maybe this was just not refactored yet

Reply:
Same, for the other side, it should show _your_ object slider

![Screenshot from 2021-04-27 23-19-48](https://user-images.githubusercontent.com/1193222/116307754-65904080-a7af-11eb-91b0-fb7fb906d9da.png)

Reply:
Yeah, i was just using it for testing my logic and forgot to remove.
Anyway, I have refactored and added stuff now to include Armando's work on percentages.

Reply:
Without the placeholder it would show all of the necessary stuff